src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,14 +1,11 @@
 <script setup>
-import { ref } from 'vue';
+import { ref,computed } from 'vue';
 
 const text = ref('');
-const onegrams = ref([]);
-const twograms = ref([]);
-const threegrams = ref([]);
 const keywords = ref([]);
-const showWords = ref(false);
+const maxNGrams = ref(10);
```

Comment:
Why `maxNGrams` is 10?

Reply:
In the task description it was written that the application should show up to 10-gram keywords.